modules/battery_manager/ajax/get_form_data.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Battery Manager adder
+ *
+ * Gets data to populate dropdowns in Add tab
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+
+
+if (isset($_GET['action'])) {
+    $action = $_GET['action'];
+    if ($action === "getFormData") {
+        echo getFormData();
+    }
+}
+
+/**
+ * Return object of fields and their values from database
+ *
+ * @return array
+ */
+function getFormData()
+{
+    $formData = [
+                 'instruments' => Utility::getAllInstruments(),
+                 'stages'      => getStagesList(),
+                 'subprojects' => Utility::getSubprojectList(null),
+                 'visits'      => Utility::getVisitList(),
+                 'sites'       => Utility::getSiteList(false),
+                 'firstVisits' => getYesNoList(),
+                ];
+
+    return json_encode($formData);
+}
+
+/**
```

Comment:
functions below seem to be duplications from the main class functions, use those instead, avoid codeduplications

Reply:
@ridz1208 Please elaborate/be more specific about which functions.